### PR TITLE
Pattern Library: Ensure RTL is taken into account for resizing patterns

### PR DESCRIPTION
**Pull request by olaseni**
Due to lack of support for RTL discovered in pdtkmj-2wZ-p2#comment-4711, pattern resizing was buggy in the affected languages.

This PR supplies a fix by ensuring that the RTL status of the locale is utilised to properly set properties for the resize box. 

Testing
---

Test the resize feature of a pattern when a RTL language like Arabic is enabled. Specify the language by using the special form of the path that supports locale e.g. `/ar/patterns/about` 

Screenshot 2024-04-03 at 17 02 59

### CODE REVIEW

File: client/my-sites/patterns/components/pattern-preview/index.tsx [hunk 1]
```diff
@@ -336,9 +337,9 @@ export function PatternPreview( props: PatternPreviewProps ) {
 		<ResizableBox
 			enable={ {
 				top: false,
-				right: true,
+				right: isRtl ? false : true,
```
Comment (nightnei) [inline client/my-sites/patterns/components/pattern-preview/index.tsx]:
```suggestion
				right: ! isRtl,
```

File: client/my-sites/patterns/components/pattern-preview/index.tsx [hunk 2]
```diff
@@ -336,9 +337,9 @@ export function PatternPreview( props: PatternPreviewProps ) {
...
 				bottom: false,
-				left: false,
+				left: isRtl ? true : false,
```
Comment (nightnei) [inline client/my-sites/patterns/components/pattern-preview/index.tsx]:
```suggestion
				left: isRtl,
```

**Review by nightnei — APPROVED** [review]
LGTM 
Screenshot 2024-04-03 at 17 37 57

File: client/my-sites/patterns/components/pattern-preview/index.tsx (hunk 1 above)
Comment (olaseni) [reply client/my-sites/patterns/components/pattern-preview/index.tsx]: Smart.

**Pull request merged**